.github/styles/Vocab/Docs/accept.txt

```diff
@@ -1,66 +1,99 @@
 ad hoc
-addon
+(Addon|addon)
 Airflow
-Alertmanager
+(Alertmanager|alertmanager)
 ANN(|s)
 Ansible
 autoscal(e|es|ed|ing)
+auto-assign
+(Auto-Provisioner|auto-provisioner)
+awscsiprovisioner
 backend
 backoff
+bootup
 Cassandra
+CentOS
 checkpoint(|s|ed|ing)
 Chronos
+clusterawsadm
 CNN(|s)
 CNTK
-configmap
-Component
 component
+Component
 Conda
-containerd
+config
+configmap
+configMap
+ConfigMap
+(Containerd|containerd)
 contravariant
 COPS
+coredns
 Couchbase
 covariant
 covariate
 cpu
+CPU
 CRD
+Crunchbase
 datacenter
+datasource
 D2iQ
+daemonsets
+Datadog
 Datastax
 DC/OS
 dcos
-Dex
+declaratively
+(Dex|dex)
 Dispatch
 DKP
 dkp
 Dockerfile
 DKLB
 DSL(|s)
 ECR
+(Elasticsearch|elasticsearch)
+elasticsearchexporter
+ETCD
 etcd
+felix
+finalizer(|s)
+fips
+FIPS
+firewalld
 Flink
-fluentbit
+(fluentbit|Fluentbit)
+Fluentd
+gapped
```

Comment:
We'll need to change 'gapped' so that it also has environment with it. So:
air gapped environment - ok
air gapped - not ok

Reply:
Done.